estuary/api/v1.py

```diff
@@ -218,3 +222,42 @@ def _get_partial_stories(query, reverse=False):
         all_results.append(rv)
 
     return jsonify(all_results)
+
+
+@api_v1.route('/siblings/<resource>/<uid>')
+def get_siblings(resource, uid):
+    """
+    Get siblings of next/previous node that are correlated to the node in question.
+
+    :param str resource: a resource name that maps to a neomodel class
+    :param str uid: the value of the UniqueIdProperty to query with
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raises NotFound: if the item is not found
+    :raises ValidationError: if an invalid resource was requested
+    """
+    # If last is True, we fetch siblings of the next node, previous node otherwise
+    last = request.args.get('last', False)
+
+    item = get_neo4j_node(resource, uid)
+    if not item:
+        raise NotFound('This item does not exist')
+
+    item_story_flow = story_flow(item.__label__)
```

Comment:
This could cause problems if the user requested a `KojiBuild` but `get_neo4j_node` returned a `ContainerKojiBuild`.

Reply:
This would not cause problems because `get_neo4j_node` uses neomodel to query Neo4j and neomodel will only return the label you requested for. Hence, the problem you mention is taken care of inherently.

Reply:
Oh alright, perfect.

Reply:
neomodel_flaws++ :p